### PR TITLE
Update slack invitation link

### LINKED ISSUE #9636: slack invitation link in README.md is expired
Get this message when I am trying to click the slack link in README.md.
```
Invitation expired
It looks like you’re trying to accept an invitation, but the link was deactivated.

Contact the person who originally invited you about getting a new link or manually inviting you.
```
Is that link broken or it dones't work for me only?

### PR DESCRIPTION
**Pull request by WT-Liu**
### What problem does this PR solve?

Issue Number: close #9636  

Problem Summary: The Slack invitation link in readme is expired.

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed: update with a new invitation link that will not expire

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- PR to update `pingcap/tidb-ansible`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### Release note 
- No release note

### CODE REVIEW

**Comment by WT-Liu** [comment]
@andylokandy @dcalvin @youjiali1995 @BusyJay Please take a look. Thanks!

File: README.md [hunk 1]
```diff
@@ -143,7 +143,7 @@ appearance, race, religion, or sexual identity and orientation.
 
 ### Slack
 
-Join the TiKV community on [Slack](https://join.slack.com/t/tikv-wg/shared_invite/enQtNTUyODE4ODU2MzI0LWVlMWMzMDkyNWE5ZjY1ODAzMWUwZGVhNGNhYTc3MzJhYWE0Y2FjYjliYzY1OWJlYTc4OWVjZWM1NDkwN2QxNDE) - Sign up and join channels on TiKV topics that interest you.
+Join the TiKV community on [Slack](https://slack.tidb.io/invite?team=tikv-wg&channel=general) - Sign up and join channels on TiKV topics that interest you.
```
Comment (BusyJay) [inline README.md:146]: How about hosting it on tikv.org? Something like https://slack.tikv.org/invite?team=tikv-wg&channel=general.

**Review by andylokandy — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (andylokandy) [reply README.md:146]: Sounds not bad. May do it later

**Review by youjiali1995 — COMMENTED** [review]
LGTM

**Comment by andylokandy** [comment]
@youjiali1995 Please use `\lgtm`

**Comment by aep** [comment]
that link is dead now

**Comment by you06** [comment]
> that link is dead now

It works now :)

**Pull request merged**